Jenkinsfile

```diff
@@ -55,7 +55,7 @@ hose {
             'env': ['ZOOKEEPER_HOSTS=%%ZOOKEEPER:2181']]],
         ['POSTGRESQL':[
                     'image': 'postgresql:9.3',
-                    'sleep': 30,
+                    'sleep': 90,
```

Comment:
@witokondoria  says that use less or equal to 60, and if you need to have the postgres running with priority, change the order in the loading of docker images